website/project/views/contributor.py

```diff
@@ -633,7 +633,6 @@ def claim_user_form(auth, **kwargs):
             # Redirect to CAS and authenticate the user with a verification key.
```

Comment:
Please make a utility/helper out of line 627 `user.verification_key = security.random_string(20)` looks like this was already begun for confirm/claim tokens @ https://github.com/CenterForOpenScience/osf.io/blob/develop/framework/auth/core.py#L45

Searching the code base this shows up in a few locations:
https://github.com/CenterForOpenScience/osf.io/search?utf8=%E2%9C%93&q=security.random_string

@abought do you have any concern or recommendation on the length/complexity of the token generated.